### PR TITLE
Fix JindoRuntime support volume mount

### PR DESCRIPTION
**Pull request by frankleaf**
### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

### CODE REVIEW

**Comment by cheyang** [comment]
@frankleaf please fix 

```
Error: pkg/ddc/jindocache/transform.go:242:11: error strings should not be capitalized (ST1005)
Error: pkg/ddc/jindofsx/transform.go:226:11: error strings should not be capitalized (ST1005)
Error: Process completed with exit code 1.
```

File: charts/jindocache/CHANGELOG.md [hunk 1]
```diff
@@ -97,4 +97,7 @@ Add support for pvc subpath feature
 Add JindoCache Engine
 
 0.9.1
-Fix JindoCache Engine sts token volume mount bug
\ No newline at end of file
+Fix JindoCache Engine sts token volume mount bug
+
+1.0.0
+Fix support master volume and set namespace meta dir
```
Comment (cheyang) [inline charts/jindocache/CHANGELOG.md:103]: I think you can remove `Fix`.

File: charts/jindocache/templates/master/statefulset.yaml [hunk 2]
```diff
@@ -186,6 +186,9 @@ spec:
               {{- end }}
           {{- end }}
           {{- end }}
+          {{- if .Values.master.volumeMounts }}
```
Comment (cheyang) [inline charts/jindocache/templates/master/statefulset.yaml:189]: I think not only master needs volumeMounts so does worker and fuse.

**Review by cheyang — CHANGES_REQUESTED** [review]

File: pkg/ddc/jindofsx/transform.go [hunk 3]
```diff
@@ -218,6 +222,11 @@ func (e *JindoFSxEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, met
 	// combine properties together
 	if len(runtime.Spec.Master.Properties) > 0 {
 		for k, v := range runtime.Spec.Master.Properties {
+			if k == "namespace.meta-dir" && runtime.Spec.Master.Replicas == 3 {
+				err = fmt.Errorf("Not support set namespace.meta-dir with %v with replicas = 3", v)
```
Comment (cheyang) [inline pkg/ddc/jindofsx/transform.go:226]: error strings should not be capitalized (ST1005)

File: charts/jindocache/CHANGELOG.md (hunk 1 above)
Comment (frankleaf) [reply charts/jindocache/CHANGELOG.md:103]: ok

File: charts/jindocache/templates/master/statefulset.yaml (hunk 2 above)
Comment (frankleaf) [reply charts/jindocache/templates/master/statefulset.yaml:189]: no need

File: pkg/ddc/jindofsx/transform.go (hunk 3 above)
Comment (frankleaf) [reply pkg/ddc/jindofsx/transform.go:226]: ok

File: pkg/ddc/jindofsx/transform.go [hunk 4]
```diff
@@ -218,6 +222,11 @@ func (e *JindoFSxEngine) transformMaster(runtime *datav1alpha1.JindoRuntime, met
 	// combine properties together
 	if len(runtime.Spec.Master.Properties) > 0 {
 		for k, v := range runtime.Spec.Master.Properties {
+			if k == "namespace.meta-dir" && runtime.Spec.Master.Replicas == 3 {
+				err = fmt.Errorf("Not Support Set NamespaceMetaDir With %v And Replicas = 3", v)
```
Comment (cheyang) [inline pkg/ddc/jindofsx/transform.go:226]: err = fmt.Errorf("it's not able to support set namespaceMetaDir with %v and replicas = 3", v)

Comment (frankleaf) [reply pkg/ddc/jindofsx/transform.go]: done

File: charts/jindocache/Chart.yaml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 5.0.0
+appVersion: 6.1.1
 version: 0.9.1
```
Comment (TrafalgarZZZ) [inline charts/jindocache/Chart.yaml:3]: Better change version: 0.9.1 to version: 1.0.0

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**